packages/owl-bot/src/copy-code.ts

```diff
@@ -196,6 +187,34 @@ export async function loadOwlBotYaml(destDir: string): Promise<OwlBotYaml> {
   return owlBotYamlFromText(text);
 }
 
+/**
+ * Clones remote repos.  Returns local repos unchanged.
+ * @param repo a full repo name like googleapis/nodejs-vision, or a path to a local directory
+ * @param workDir a local directory where the cloned repo will be created
+ * @param logger a logger
+ * @param depth the depth param to pass to git clone.
+ * @returns the path to the local repo.
+ */
+export function toLocalRepo(
```

Comment:
I like the introduction of this helper to encapsulate some of the git logic.

Reply:
> This logic looks good to me, I believe you said you were following on with some tests though?

Yes.  I added tests in https://github.com/googleapis/repo-automation-bots/pull/1488.  Let's continue the review there, and I'll close this PR.